signaling/develop/authentication.mdx

```diff
@@ -34,15 +34,17 @@ a <Vg k="SIG" /> token is a dynamic key generated on your app server that is val
 
 In order to follow this procedure you must have the following:
 
--   A valid [<Vg k="COMPANY" /> account](../../{local_product}/reference/manage-agora-account.xml#_create_an_agora_account).
+-   A valid [<Vg k="COMPANY" /> account](../reference/manage-agora-account#_create_an_agora_account).
 
--   An Agora project with the [App Certificate](../../{local_product}/reference/manage-agora-account.xml#_get_the_app_certificate) enabled.
+-   An Agora project with the [App Certificate](../reference/manage-agora-account#_get_the_app_certificate) enabled.
 
 -   [Golang](https://golang.org/) 1.14+ with GO111MODULE set to on.
 
 -   If you are using Go 1.16+, GO111MODULE is on by default. See [this blog](https://blog.golang.org/go116-module-changes) for details.
 
--   [npm](https://www.npmjs.com/get-npm) and a [supported browsers](../../{local_product}/reference/supported-platforms.xml#supported_platform_browsers).
+-   [npm](https://www.npmjs.com/get-npm) and a [supported browsers](../reference/supported-platforms#supported_platform_browsers).
+
+- For AccessToken2, <Vg k="MESS_SDK" /> v1.5.0 or  greater.
```

Comment:
You better keep the previous auth doc of Token 006 in Reference or somewhere else.
<img width="921" alt="image" src="https://user-images.githubusercontent.com/101544891/192248578-dd6665ee-a93c-416d-8112-b226e194960b.png">

Reply:
Both are in the authentication guide.